pkg/apiserver/registry/datapackaging/package_crd_rest_test.go

```diff
@@ -31,7 +35,10 @@ func TestPackageVersionListIncludesGlobalAndNamespaced(t *testing.T) {
 	internalClient := fake.NewSimpleClientset(globalIntPackageVersion(), namespacedIntPackageVersion(), excludedNonGlobalIntPackageVersion())
 	fakeCoreClient := k8sfake.NewSimpleClientset(namespace())
 
-	pkgvCRDREST := datapkgreg.NewPackageCRDREST(internalClient, fakeCoreClient, globalNamespace)
+	log := zap.New(zap.UseDevMode(false)).WithName("package_crd_rest_test")
+	logf.SetLogger(log)
+	klog.SetLogger(log)
+	pkgvCRDREST := datapkgreg.NewPackageCRDREST(internalClient, fakeCoreClient, globalNamespace, log)
```

Comment:
I guess we can directly use logf.Log here as we are not testing the logging part.
Same for other places.

Reply:
logger changed for the tests.